### DISCUSSION TITLE
Solo SB260-110

### DISCUSSION THREAD

**Discussion by kasibarndoor** [GerryDazoo/Slinger#323] [General]
I have read reports that the Solo works and others that it doesn't if the server and the sling box are on the same LAN and only the server has access to the WAN will a 260 work?

**Comment by GerryDazoo** [comment]
Who ever said it doesn't work if the server and slingbox are on the same LAN is wrong.

**Reply by kasibarndoor** [reply]
Sorry poor punctuation it should have read "I have read reports that the Solo works and others that it doesn't.  If the server and the sling box are on the same LAN and only the server has access to the WAN will a 260 work?"

The idea was to sit the server and sling box behind a router or firewall so that the Solo could not talk to Mr Sling and become bricked.

**Reply by GerryDazoo** [reply]
Yes that will work

**Reply by sp222222** [reply]
I have a refurb solo running with custom password and custom port using block rules on router. only open for slinGER to run its thing.
